src/system.h

```diff
@@ -79,7 +79,13 @@
 #  define SUPPORT_JOYSTICK
 #  define SUPPORT_JOYSTICK_HAT
 #  define SUPPORT_JOYSTICK_AXIS
-#else // Everything not catched above, e.g. Linux/*BSD/macOS
+#elif defined(USE_LIBRETRO)
+#  define SUPPORT_KEYBOARD
+#  define SUPPORT_JOYSTICK
+#  define SUPPORT_JOYSTICK_HAT
+#  define SUPPORT_JOYSTICK_AXIS
+#  define JOYSTICK_AXIS_SENSIBILITY 20000
+#else // Everything not catched above, e.g. Linux/*BSD/macOS/libretro
```

Comment:
libretro has an own section above.

Reply:
Wups, guess you can imagine based on this comment how the code evolved over time :D